avocado/core/nrunner.py

```diff
@@ -460,12 +460,18 @@ def run(self):
 
         if env and 'PATH' not in env:
             env['PATH'] = os.environ.get('PATH')
-        process = subprocess.Popen(
-            [self.runnable.uri] + list(self.runnable.args),
-            stdin=subprocess.DEVNULL,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-            env=env)
+        try:
+            process = subprocess.Popen(
+                [self.runnable.uri] + list(self.runnable.args),
+                stdin=subprocess.DEVNULL,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                env=env)
+        except Exception as e:
+            yield self.prepare_status('started')
```

Comment:
I think we should send also stderr message to save the traceback for easier debugging. 
Something like:
`self.prepare_status('running', {'type': 'stderr', 'log': traceback.format_exc()})`

Reply:
I don't like much this idea, because gives the impression that the script actually started running. And again we are mixing test output with internal Avocado traceback.